CRM/Admin/Form/SettingTrait.php

```diff
@@ -196,13 +196,13 @@ protected function addFieldsDefinedInSettingsMetadata() {
           $this->$add(
             $props['html_type'],
             $setting,
-            ts($props['title']),
+            $props['title'],
```

Comment:
This may break things, but it's up to the extension to call `ts` in their setting file. If this sometimes works, it's only because the string is already in CiviCRM elsewhere.